### DISCUSSION TITLE
nx migrate latest and angular releases?

### DISCUSSION THREAD

**Discussion by leon** [nrwl/nx#8095] [General]
NX 13.3 was just released

and Angular 13.1 was just released.

but when I run `nx migrate latest` I'm just getting the NX release.

I think this is because NX pins the angular release for each version of NX and in this case we where unfortunate that angular 13.1 was released just after NX 13.3

But I really want to update to Angular 13.1, what are my options here?

Can I force update to the latest Angular using the nx cli?

**Comment by avchugaev** [comment]
@leon I had the same situation. The more I discover monorepo concept, the more flaws I see. This is one them: the bigger monorepo becomes, the harder to manage its third-party dependencies. For instance, I use both Angular and NestJS in the same monorepo. But I soon realized that in such case, conflicts among third-party dependencies (and transient dependencies) are inevitable. For instance, Angular and NestJS may require different versions of TypeScript or RxJS. On one hand, it's nice NX resolves these conflicts and provides migrations, but it still an issue to wire fullstack project's deps. This means we will experience delays between the newest frameworks releases and their availability in NX.

I suppose, force-update of frameworks/libraries managed by NX may broke the consistency and deps graph stability.

You should decide whether such delays are okay for you, or having the latest version of framework(s) at the release-date is a must-have.

Everything has its price.
